src/content/docs/apm/agents/java-agent/getting-started/migration-8x-guide.mdx

```diff
@@ -15,7 +15,7 @@ This guide covers the major changes between the 7.x and 8.x series of the Java a
 * [Removed instrumentation](#removed_instrumentation)
 
 <Callout variant="important">
-In version 8.8.0 the attributes removed in 8.x - 8.7x were reintroduced for backward compatibility for customers that still have alerts or dashboards using which use them.
+In version 8.8.0 the attributes removed in 8.x - 8.7x were reintroduced for backward compatibility for customers that still have alerts or dashboards that use them.
```

Comment:
Per our style guide, we aren’t supposed to use the .x styling for version numbers: https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#version

What are the exact versions that this covers?